src/Web/Sfa.Das.Sas.ApplicationServices/ApprenticeshipSearchService.cs

```diff
@@ -2,7 +2,6 @@
 using Sfa.Das.Sas.ApplicationServices.Logging;
 using Sfa.Das.Sas.ApplicationServices.Models;
 using Sfa.Das.Sas.ApplicationServices.Settings;
-
```

Comment:
Resharper suggests reordering the using statements. The suggestion is:

```
namespace Sfa.Das.Sas.ApplicationServices
{
    using System.Collections.Generic;
    using Logging;
    using Models;
    using Settings;
    using SFA.DAS.NLog.Logger;
```

Reply:
yeah, done that now.